fenix/app/src/main/java/org/mozilla/fenix/tabstray/inactivetabs/InactiveTabs.kt

```diff
@@ -105,6 +107,7 @@ fun InactiveTabsList(
                         FaviconListItem(
                             label = tab.toDisplayTitle(),
                             description = tabUrl,
+                            faviconPainter = tab.content.icon?.run { BitmapPainter(this.asImageBitmap()) },
```

Comment:
Related to my comment above about Favicon painters:

I'm just discovering this, but it seems odd to me that we are keeping this `tab.content.icon` in memory _and_ we have a separate memory cache for icons through the `components.core.icons.Loader()` function tree. I think it could be worth investigating in a follow-up if we're doing unnecessary work.

Reply:
yeah, it's a good question of whether or not they are living in two places simultaneously. from a cursory search, it looks like the tab icons are being set via an "Icons" extension callback and then browser icons maintains a cache but because of JVM magic, they could be the same thing in memory with two references? Not entirely sure, but this would require some more thorough investigation,.